third_party/libtock-drivers/src/result.rs

```diff
@@ -0,0 +1,133 @@
+use core::fmt;
+
+pub use libtock_core::result::*;
+
+pub type TockResult<T> = Result<T, TockError>;
+
+// We sometimes need to handle errors in a `TockResult` by calling `unwrap`. However,
+// `Result::unwrap` requires that the error type implements `core::fmt::Debug`. Under the hood,
+// this requires dynamic dispatch, which has non-negligible overhead on code size. Therefore errors
+// don't derive from `Debug` in libtock-rs.
+//
+// Instead one can call `.ok().unwrap()` which relies on `Option::unwrap` and doesn't require any
+// debugging of the error type.
+//
+// This trait allows to flexibly use `Result::unwrap` or `Option::unwrap` and is configured to do
+// so depending on the `debug_ctap` feature.
```

Comment:
Now I'm confused.
Who authored `third_party/libtock-drivers`? According to the TOML it's Tock team bug `debug_ctap` is OpenSK specific. Is it Google's code, Tock's code or a mix of both?

Reply:
It's a mix of both. It is derived from the libtock-rs developed by "Tock Project Developers", with some additional code (in this pull request) to make it work with the new syscall API.

In a sense "Tock Project Developers" already includes some Google employees that contributed to (lib)tock, but if that's too unclear, I can add explicit names in the `authors` section.